.travis.yml

```diff
@@ -62,7 +62,8 @@ jobs:
       arch: amd64
       before_script:
         - go get github.com/onsi/ginkgo/ginkgo
-        - go get sigs.k8s.io/kind@v0.7.0
+        - go get sigs.k8s.io/kind
```

Comment:
I think we should use the fixed version of kind, so we will know the k8s version in CI.